platforms/hyperledger-indy/charts/indy-auth-job/templates/job.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
```

Comment:
What I'm thinking about is how we'll get the jwt reviewer token token for a different serviceaccount than this job will use (the job will use the admin sa and create vault auth for a different one)...

Reply:
Can we setup the service account to allow us to use "kubectl" inside of pod and then just get the token as we do it in roles?
https://stackoverflow.com/questions/42642170/how-to-run-kubectl-commands-inside-a-container
this example worked for me and I was able to run kubectl and get resources.
Not sure if that is a good practise or if its safe to do.